tests/vlan/test_autostate_disabled.py

```diff
@@ -10,6 +10,25 @@
 ]
 
 
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthosts, rand_one_dut_hostname, loganalyzer):
+    """
+       Ignore expected errors in logs during test execution
+
+       Args:
+           loganalyzer: Loganalyzer utility fixture
+           duthost: DUT host object
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    if loganalyzer:
+        loganalyzer_ignore_regex = [
```

Comment:
Is this fix for all platform or only E1031? If only E1031, I think it's better to check hwsku from `duthost` first.